src/components/RightPane/CoursePane/SearchForm/AdvancedSearch.tsx

```diff
@@ -59,7 +83,7 @@ class AdvancedSearchTextFields extends PureComponent {
         });
     };
 
-    handleChange = (name) => (event) => {
+    handleChange = (name: string) => (event: ChangeEvent<{ checked?: boolean, name?: string | undefined; value: unknown; }>) => {
```

Comment:
pretty sure `string|undefined` is redundant if you have the `?` on the parameter to make it optional.